### PR TITLE
content/agent: added an introduction + installation guide

### PR DESCRIPTION
**Pull request by odeke-em**
Firstly an introduction and installation guide for
the OpenCensus Agent aka "ocagent".

Updates #560

### CODE REVIEW

File: content/agent/_index.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+can at anytime decide which backend they'd like to export to.
+An example of such a provider is [Microsoft with Azure Application Insights.](https://docs.microsoft.com/en-us/azure/azure-monitor/app/opencensus-local-forwarder)
+which has their own agent that receives OpenCensus observability signals.
+
+* Enabling exporters in only one place dramatically reduces the need to restart and redeploy each application that produces observability.
+Only ocagent needs to be restarted and the other applications can stay deployed in production.
+
+* To bootstrap and democratize collection and exporting of your observability data, ocagent also comes with "receivers". "receivers" make ocagent act like
+pass-through backends that then receive your telemetry and then route it to the various exporter of your choice. For example if all your applications used Zipkin or Jaeger or Prometheus,
+ocagent can receive data from your applications and then route it to your target backend. This is particulary important if you are in a polyglot and poly-backend setup
```
Comment (songy23) [inline content/agent/_index.md]: s/particulary/particularly

File: content/agent/_index.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+enable the OpenCensus Agent exporter for their target language and it will deliver stats, metrics and traces
+to the centralized point before they are exported to the target backends.
+
+By default, ocagent runs on TCP port `55678` and receives traffic from:
+
+* HTTP/2 clients with Protobuf messages
+* HTTP/1 clients that use the [grpc-web-gateway](https://github.com/grpc-ecosystem/grpc-gateway)
+
+ocagent is written in the [Go programming language](https://golang.org/), it is cross platform, self-monitored and receives traffic
+from any application that supports the [ocagent protocol](https://github.com/census-instrumentation/opencensus-proto/tree/master/src/opencensus/proto/agent) regardless of the programming language and deployment.
```
Comment (songy23) [inline content/agent/_index.md]: More precisely, Agent can receive traffic from applications instrumented with libraries that the receivers supported, like Zipkin/Jaeger/Prometheus. Supporting the ocagent protocol shouldn't be a requirement here.

File: content/agent/_index.md (hunk 1 above)
Comment (odeke-em) [reply content/agent/_index.md]: Thank you, I am updating it!

File: content/agent/_index.md (hunk 2 above)
Comment (odeke-em) [reply content/agent/_index.md]: That can be added about receivers(that's a different point), but the point here is that if your language/library supports the agent protocol, then you can use it with the agent.

Comment (odeke-em) [reply content/agent/_index.md]: Also, I plan on adding a section for receivers hence why I won't be providing a URL :)

**Comment by odeke-em** [comment]
Thank you for the review @songy23!

**Pull request merged**